test/integration/running/test_running_cluster.py

```diff
@@ -45,6 +45,18 @@ def wait_cluster_started(tt_cmd, workdir, app_name, instances, inst_conf):
         files.append(conf['console_socket'])
     assert wait_files(5, files)
 
+    def is_bootstrap_completed():
+        status_cmd = [tt_cmd, "status", app_name]
+        status_rc, status_out = run_command_and_get_output(status_cmd, cwd=workdir)
+        assert status_rc == 0
+        status_info = extract_status(status_out)
+        for inst in instances:
+            inst_id = app_name + ":" + inst
+            if "BOX" not in status_info[inst_id] or status_info[inst_id]["BOX"] != "running":
+                return False
+        return True
```

Comment:
As I can see, the `running` state of `box.info.status` means that the instance is loaded. Therefore this name of function is not only about bootstrap condition. Maybe we can rename it somehow, `is_cluster_loaded` for example?

Reply:
You are right, the name is not proper. Fixed it (I've renamed it to just `are_all_box_statuses_running` because in fact it just checks that all box statuses are running).